src/TestProblem/Hydro/CCSN/Init_TestProb_Hydro_CCSN.cpp

```diff
@@ -568,15 +577,56 @@ void Record_CCSN()
          Record_CCSN_GWSignal();
          DumpTime_CCSN += CCSN_GW_DT;
       }
-   }
+   } // if ( CCSN_GW_OUTPUT )
 #  endif
 
+
+// (3) Check whether the core bounce occurs
+   if ( !CCSN_Is_PostBounce )
+   {
+      Detect_CoreBounce();
+
+      if ( CCSN_Is_PostBounce )
+      {
+//       dump the bounce time in standard output
+         if ( MPI_Rank == 0 )   Aux_Message( stdout, "Bounce time = %13.7e !!\n", Time[0] * UNIT_T );
```

Comment:
since your have converted the unit, it might nice to include the unit in the print message?  (e.g. `"Bounce time = %13.7e ms !!\n"`)

Reply:
True. I will udpate it in next commit.